opentelemetry-sdk/src/opentelemetry/sdk/_metrics/view.py

```diff
@@ -0,0 +1,172 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# pylint: disable=protected-access
+
+from dataclasses import dataclass
+from logging import getLogger
+from re import match
+from typing import Dict
+
+from opentelemetry._metrics.instrument import Instrument
+from opentelemetry.sdk._metrics.aggregation import Aggregation
+
+_logger = getLogger(__name__)
+
+
+@dataclass
+class View:
+    def __init__(
+        self,
+        instrument_type: Instrument = None,
+        instrument_name: str = None,
+        meter_name: str = None,
+        meter_version: str = None,
+        meter_schema_url: str = None,
+        name: str = None,
+        description: str = None,
+        attribute_keys: Dict[str, str] = None,
+        aggregation: Aggregation = None,
```

Comment:
I don't think we should expose the aggregation classes and let users pass in custom aggregations yet, as I have a feeling it may be added to the spec later on. Can we just use an enum for this in the meantime?

Reply:
Well, this is still private but if we use an enum, then release we would be stuck with an enum, it would be a breaking change to use a class later.

Reply:
> Well, this is still private

I don't follow, how is it private?

> it would be a breaking change to use a class later.

Good point. Can you check what other language SDKs are doing?

Reply:
Apparently Java uses an aggregation class argument as well: https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/view/View.java#L32

Reply:
If you look close, that aggregation class is an `io.opentelemetry.sdk.metrics.view.Aggregation` and it's job is to map to a `io.opentelemetry.sdk.metrics.internal.aggregator.Aggregator`. So they don't expose the actual aggregation interface

Reply:
I have updated this to receive an aggregation instance instead of a class.